src/index.html

```diff
@@ -8,6 +8,49 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars-block">
+      <ul class="stars stars--0">
+        <li class="stars__star"></li>
+        <li class="stars__star"></li>
+        <li class="stars__star"></li>
+        <li class="stars__star"></li>
+        <li class="stars__star"></li>
+      </ul>
+      <ul class="stars stars--1">
+        <li class="stars__star"></li>
+        <li class="stars__star"></li>
+        <li class="stars__star"></li>
+        <li class="stars__star"></li>
+        <li class="stars__star"></li>
+      </ul>
```

Comment:
```suggestion
      <ul class="stars stars--0">
        <li class="stars__star"></li>
        <li class="stars__star"></li>
        <li class="stars__star"></li>
        <li class="stars__star"></li>
        <li class="stars__star"></li>
      </ul>
      
      <ul class="stars stars--1">
        <li class="stars__star"></li>
        <li class="stars__star"></li>
        <li class="stars__star"></li>
        <li class="stars__star"></li>
        <li class="stars__star"></li>
      </ul>
```

Add blank line between multiple sibling blocks of html